### PR TITLE
 feature - ui bill screen

### PR DESCRIPTION
**Pull request by at-phatnguyen**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Bill/BillViewController.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        super.viewDidLoad()
+        configNaviBar()
+    }
+
+    func configNaviBar() {
+        billImageView.layer.masksToBounds = true
+        billImageView.layer.borderColor = UIColor.white.cgColor
+        billImageView.layer.borderWidth = 3
+        billImageView.layer.cornerRadius = 10
+        title = "Bill"
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Bill/BillViewController.swift]: Define em

File: MyApp/View/Controllers/Bill/BillViewController.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    @IBAction func confirmButtonTouchUpInside(_ sender: Any) {
+        print("confirmed")
+        alert(msg: "Đã Lưu", buttons: ["OK"], handler: nil)
```
Comment (at-truongnguyen) [inline MyApp/View/Controllers/Bill/BillViewController.swift]: Define ra em

File: MyApp/Define/String.swift [hunk 3]
```diff
@@ -32,5 +32,8 @@ extension App {
         static let forgotPass = "Forgot Password"
         static let login = "Login"
         static let cart = "Cart"
+        static let saved = "Saved"
+        static let bill = "Bill"
+
```
Comment (at-daiho) [inline MyApp/Define/String.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Pull request merged**